src/SignatureBES.cpp

```diff
@@ -316,8 +316,8 @@ SignatureBES::SignatureBES(istream &sigdata, BDoc *bdoc)
         const SignedPropertiesType::SignedDataObjectPropertiesOptional &sdop = sp->signedDataObjectProperties();
         if(sdop.present())
         {
-            if(!sdop->commitmentTypeIndication().empty())
-                THROW("CommitmentTypeIndicationType is not supported");
+            /*if(!sdop->commitmentTypeIndication().empty())
```

Comment:
Commiting commented out code is discouraged. Having some debug line instead of throwing might be a way forward ?